src/components/SidebarFilter/SidebarFilter.tsx

```diff
@@ -16,6 +17,7 @@ type SidebarFilterProps = {
   onChange: (value: string) => void;
   onClear: () => void;
   onTagRemove: (value: string) => void;
+  skipLink: string;
```

Comment:
Should this be a boolean prop?

Reply:
I am passing here the `href` to SkipLink, so:

```javascript
export const SidebarFilter = ({
  skipLink,
  ...
}) => {
  return (
    <div className={classNames}>
      <SkipLink href={skipLink} text="Skip to results" />
      <header className="cc-sidebar-filter__header">...</header>
      </div>
```

Reply:
Ahh okay, how about `skipLinkHref` to make it slightly clearer?